jumpscale/packages/tfgrid_solutions/actors/solutions.py

```diff
@@ -110,6 +115,10 @@ def update_pool_farm_cache(cache_dict, pool, farm):
 
     @actor_method
     def cancel_workload(self, wid) -> bool:
+        workload = j.sals.zos.get().workloads.get(wid)
+        # Delete public ip
+        if workload.public_ip:
```

Comment:
only k8s workload have the `public_ip` attr this will cause every other workload type to fail.